com.unity.netcode.gameobjects/Runtime/Serialization/BytewiseUtility.cs

```diff
@@ -0,0 +1,80 @@
+using Unity.Collections.LowLevel.Unsafe;
+
+namespace Unity.Multiplayer.Netcode
+{
+    public static class BytewiseUtility
+    {
+        /// <summary>
+        /// Helper function optimized for quickly copying small numbers of bytes.
+        /// Faster than UnsafeUtil.Memcpy and other alternatives for amount <= 8
```

Comment:
I'd also lean towards using Unity's MemCpy instead of crafting one for the reasons Andrew outlined above. Maybe just to add a little bit on that, it's more costly to modify/change/maintain home brew solutions in the mid/long term and it always becomes a "should we throw away this thing that we invested X amount of time & effort" compared to just sticking with what's already out there in Unity core APIs.

Reply:
Removed.